docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Configuring out-of-order samples ingestion"
+menuTitle: "Configuring out-of-order samples ingestion"
+description: "Learn how to configure Grafana Mimir to handle out-of-order samples ingestion."
+weight: 120 
+---
+
+# Configuring out-of-order samples ingestion
+
+By default Grafana Mimir will only accept new samples that are consequent to the existing ingested samples for a known series. Those that are older than the last sample that was ingested are called out-of-order samples.
```

Comment:
I looked for the meaning of `consequent` and it felt like not the right place. Suggested some rewording.
```suggestion
By default Grafana Mimir will only accept new samples that are newer than the last ingested samples for a series. Those that are older than the last sample that was ingested are called out-of-order samples.
```